Project/Project/Project.csproj

```diff
@@ -13,6 +13,11 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.AspNetCore.Identity" Version="2.2.0" />
+    <PackageReference Include="MySql.Data" Version="8.4.0" />

```

Comment:
is this package really needed, I doubt cause the recommendations model works without